lib/internal/streams/utils.js

```diff
@@ -1,15 +1,16 @@
 'use strict';
 
 const {
-  Symbol,
   SymbolAsyncIterator,
+  SymbolFor,
   SymbolIterator,
 } = primordials;
 
-const kDestroyed = Symbol('kDestroyed');
-const kIsErrored = Symbol('kIsErrored');
-const kIsReadable = Symbol('kIsReadable');
-const kIsDisturbed = Symbol('kIsDisturbed');
+const kIsDestroyed = SymbolFor('node.stream.__PRIVATE__kIsDestroyed');
```

Comment:
Though they are only semi-public, these symbols do set a precedent, so would be a good time to choose between the `nodejs` prefix (like `Symbol.for('nodejs.util.inspect.custom')` and `node` (like the WinterCG [runtime key](https://github.com/wintercg/runtime-keys/pull/2)).

Reply:
sticking with nodejs.<coremodule>.whatever seems safer, since the existing nodejs ones have been around for awhile, and code likely depends on them.

Reply:
> code likely depends on them

Yes, and a search for that led me to the guidelines introduced in https://github.com/nodejs/node/pull/22684:

https://github.com/nodejs/node/blob/cd4f4b4dfac4d96316058e12aafdc2737ed43f72/doc/contributing/using-symbols.md?plain=1#L67-L68